operator/bundle_helpers/rewrite.py

```diff
@@ -14,16 +16,19 @@ def rewrite(d, rewriter):
         for i, elem in enumerate(d):
             res = rewrite(elem, rewriter)
             if res is not None:
-                list[i] = res
```

Comment:
Wow. Was there a bug in this `list[i] = res` line previously?

Reply:
I think so. But this line was never used